test/functional/apps/dashboard/group6/dashboard_esql_chart.ts

```diff
@@ -64,7 +64,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await dashboard.waitForRenderComplete();
 
       await monacoEditor.setCodeEditorValue('from logstash-* | stats maxB = max(bytes)');
-      await testSubjects.click('TextBasedLangEditor-run-query-button');
+      await testSubjects.click('ESQLEditor-run-query-button');
```

Comment:
Would it make sense to move this to a page object or service?

Reply:
It would but not on this PR.